components/hab/src/command/studio/docker.rs

```diff
@@ -219,8 +227,35 @@ where
     Ok(())
 }
 
+fn container_exists(docker_cmd: &Path) -> bool {
+    let name = path_to_name().expect("Unable to load name from path");
+    let cmd_args: Vec<OsString> = vec!["container".into(), "inspect".into(), name.into()];
+    let output = Command::new(docker_cmd)
+        .args(&cmd_args)
+        .output()
+        .expect("docker failed to start");
+    if output.status.success() {
+        return true;
+    }
+    return false;
+}
+
+fn path_to_name() -> Result<String> {
+    let mut cwd = env::current_dir()?;
+    cwd = cwd.strip_prefix("/")?.to_path_buf();
+    let pathstr = cwd.to_str().expect("Path to be parseable");
+    return Ok(pathstr.replace("/", "--"));
+}
```

Comment:
I don't think this will work on Windows. You may want to replace backslashes with slashes first and then strip *up to* the first `/`.

Reply:
Would it also work to make this function a cfg target since all the path logic is isolated here?